scripts/post_install.js

```diff
@@ -0,0 +1,97 @@
+// Copyright 2018. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+'use strict';
+
+const path = require('path');
+const process = require('process');
+const { spawn } = require('child_process');
+
+//
+// Path constants.
+//
+
+const SCRIPT_DIR = __dirname;
+const SOURCE_ROOT = path.join(SCRIPT_DIR, '..');
+
+//
+// Platform vars.
+//
+
+const isWinPlatform = ('win32' === process.platform);
+
+/**
+ * Executes any arbitrary shell commands.
+ * 
+ * @param {string} cmd The command to execute. 
+ * @param {...string} arg Optional arguments for the provided command. 
+ */
+function runCommand(cmd, ...args) {
```

Comment:
We only need this once, and the code will only get more complex if we pass promises around. Could we make it `runCommandThenExit`?

Reply:
ok